### PR TITLE
Update library services menu

### LINKED ISSUE #1444: Update library services menu
Screenshot and HTML below. Notes:

1. would be helpful to be able to update this relatively easily. Otherwise, it goes stale and we ignore.
2. note current mix of http and https. Fall goal for website is to switch to https, but not sure when this will be complete. For staff, if they are currently logged in, using https will show them the Drupal menus - so I'd like to avoid this for now.
3. HTML snippet below is just to provide copiable text and URLs, doesn't assume anything about classes, roles, etc. - see existing code for whatever that's doing.
4. All nested options are visible - it's not expand/collapse.

library services menu

```
<ul>
  <li>
    <a href="http://library.stanford.edu/using">Using the libraries</a>
    <ul>
        <li>
        <a href="http://library.stanford.edu/using/connect-campus">Connecting to e-resources</a>
        </li><li>
        <a href="http://library.stanford.edu/using/interlibrary-borrowing">Interlibrary services</a>
        </li>
    </ul>  
  </li><li>
    <a href="http://library.stanford.edu/ask">Ask us</a>
    <ul>
        <li>
        <a href="http://library.stanford.edu/ask/email/reference">Ask a reference question</a>
        </li><li>
        <a href="http://library.stanford.edu/ask/email/connection-problems">Report a connection problem</a>
        </li><li>
        <a href="https://web.stanford.edu/group/dlss/requestform/suggestpurchase.fb">Suggest a purchase (requires login)</a>
        </li>
    </ul> 
  </li><li>
    <a href="http://library.stanford.edu/about">About the Stanford Libraries</a>
    <ul>
        <li>
        <a href="http://library.stanford.edu/people">Find a specialist</a>
        </li><li>
        <a href="https://library-hours.stanford.edu/">Library hours</a>
        </li>
    </ul>
  </li><li>
    <a href="http://library.stanford.edu/libraries">Libraries</a>
  </li><li>
    <a href="http://library.stanford.edu/collections">Collections</a>
  </li><li>
    <a href="http://library.stanford.edu/research">Research support</a>
    <ul>
        <li>
        <a href="http://library.stanford.edu/guides/topic">Topic guides</a>
        </li><li>
        <a href="http://library.stanford.edu/guides/course">Course guides</a>
        </li><li>
        <a href="http://library.stanford.edu/search-tools">More search tools</a>
        </li>
    </ul>  
  </li>
</ul>
```

### PR DESCRIPTION
**Pull request by camillevilla**
Fixes #1444 

This PR:
- updates the link content of the Library Services menu
- increases the width of the menu to 300px and adjusts submenu padding
- renames `search_navbar_top_menu` to `search_navbar_services_menu` in several files to reflect its place in the redesign

### Before
services-menu-before

### After
services-menu-after

### CODE REVIEW

File: app/assets/stylesheets/modules/subnavbar.scss [hunk 1]
```diff
@@ -25,7 +25,15 @@
     color: $white;
   }
 
+  .sul-services-menu {
+    min-width: 300px;
+  }
+
   // For all subnavbar navbar lists
+  ul ul ul {
```
Comment (jcoyne) [inline app/assets/stylesheets/modules/subnavbar.scss]: Should/could any of these tags be qualified (with a class)?

File: app/assets/stylesheets/modules/subnavbar.scss [hunk 2]
```diff
@@ -25,7 +25,15 @@
     color: $white;
   }
 
+  .sul-top-menu {
+    min-width: 300px;
```
Comment (jcoyne) [inline app/assets/stylesheets/modules/subnavbar.scss]: Could this  be done with padding instead?  Is 300px going to be correct for all zoom levels and screensizes?

File: app/assets/stylesheets/modules/subnavbar.scss [hunk 3]
```diff
@@ -25,7 +25,15 @@
     color: $white;
   }
 
+  .sul-services-menu {
+    min-width: 300px;
```
Comment (jcoyne) [inline app/assets/stylesheets/modules/subnavbar.scss]: Could this be done with padding instead? Is 300px going to be correct for all zoom levels and screen sizes?

Comment (camillevilla) [reply app/assets/stylesheets/modules/subnavbar.scss]: @jcoyne I discovered `white-space: nowrap` and gave that a try. At smaller screen sizes / higher zoom levels, the menu dropdown is 100% wide and `white-space` is reset to `normal`. What do you think?

File: app/views/shared/_search_navbar_services_menu.html.erb [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<ul class="dropdown-menu sul-services-menu" role="menu" aria-labelledby="searchNavbarTopMenu">
+  <li>
+    <a role="menuitem" href="https://library.stanford.edu/using">Using the libraries</a>
+    <ul>
+        <li>
+        <a role="menuitem" href="https://library.stanford.edu/using/connect-campus">Connecting to e-resources</a>
```
[the remark this answers is not in the document]
Comment (camillevilla) [reply app/views/shared/_search_navbar_services_menu.html.erb]: @mejackreed hmm, strange. I've fixed it up and put it through `atom-beautify`. Let me know if that's improved (and if you have any recommendations for linters!).

[the remark this answers is not in the document]
Comment (mejackreed) [reply app/views/shared/_search_navbar_services_menu.html.erb]: 👍 Looks great!

File: app/assets/stylesheets/modules/subnavbar.scss (hunk 1 above)
Comment (camillevilla) [reply app/assets/stylesheets/modules/subnavbar.scss]: @jcoyne thanks for this one, I've updated it to `.sul-services-menu ul` to better capture the idea that it's a sublist for a specific menu.

**Comment by mejackreed** [comment]
👍

**Pull request merged**